edb/schema/constraints.py

```diff
@@ -268,14 +268,25 @@ def as_alter_delta(
         self_schema: s_schema.Schema,
         other_schema: s_schema.Schema,
         context: so.ComparisonContext,
-    ) -> sd.Command:
+    ) -> sd.ObjectCommand[Constraint]:
         return super().as_alter_delta(
             other,
             self_schema=self_schema,
             other_schema=other_schema,
             context=context,
         )
 
+    def as_delete_delta(
```

Comment:
Is this needed, if it just makes a super call?

Reply:
mypy wants this for some reason